utils/src/keiyoushi/utils/Json.kt

```diff
@@ -19,3 +20,9 @@ inline fun <reified T> String.parseAs(json: Json = jsonInstance): T =
  */
 inline fun <reified T> Response.parseAs(json: Json = jsonInstance): T =
     json.decodeFromStream(body.byteStream())
+
+/**
+ * Serializes the object to a JSON String.
+ */
+inline fun <reified T> T.toJson(json: Json = jsonInstance): String =
```

Comment:
```suggestion
inline fun <reified T> T.toJsonString(json: Json = jsonInstance): String =
```
make a separate pr for this please